From 0dc48ff1f333c689664c8085a9a686b5c982bab5 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Fri, 20 May 2022 08:20:53 +0300
Subject: [PATCH 40/50] fdts: s32cc: Correct SerDes simple_bus_reg warning

This corrects dtc warnings:
Warning (simple_bus_reg): /soc/serdes@40480000: simple-bus unit address
format error, expected "40400000"
Warning (simple_bus_reg): /soc/serdes@44180000: simple-bus unit address
format error, expected "44100000"

Issue: ALB-8823
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 fdts/s32cc.dtsi | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/fdts/s32cc.dtsi b/fdts/s32cc.dtsi
index 85f8c6689..a7f0b36a3 100644
--- a/fdts/s32cc.dtsi
+++ b/fdts/s32cc.dtsi
@@ -798,9 +798,9 @@
 
 		serdes0: serdes@40480000 {
 			compatible = "nxp,s32cc-serdes";
-			reg = <0x00 0x40400000 0x0 0x80000   /* dbi registers */
-			       0x00 0x40480000 0x0 0x04000>; /* serdes registers */
-			reg-names = "dbi", "ss";
+			reg = <0x00 0x40480000 0x0 0x04000  /* serdes registers */
+			      0x00 0x40400000 0x0 0x80000>; /* dbi registers */
+			reg-names = "ss", "dbi";
 			#address-cells = <3>;
 			#size-cells = <2>;
 			device_type = "pci-generic";
@@ -845,9 +845,9 @@
 
 		serdes1: serdes@44180000 {
 			compatible = "nxp,s32cc-serdes";
-			reg = <0x00 0x44100000 0x0 0x80000   /* dbi registers */
-			       0x00 0x44180000 0x0 0x04000>; /* serdes registers */
-			reg-names = "dbi", "ss";
+			reg = <0x00 0x44180000 0x0 0x04000   /* serdes registers */
+			       0x00 0x44100000 0x0 0x80000>; /* dbi registers */
+			reg-names = "ss", "dbi";
 			#address-cells = <3>;
 			#size-cells = <2>;
 			device_type = "pci-generic";
-- 
2.17.1

